lib/sshutils/server.go

```diff
@@ -631,6 +631,8 @@ func (s *Server) HandleConnection(conn net.Conn) {
 				}(span)
 			} else {
 				span.End()
+				// Reject all global requests if there is not a handler present.
+				req.Reply(false, nil)
```

Comment:
As we're entering this brand new world where we don't violate the SSH spec, can I bother you with adding a couple of `go ssh.DiscardRequests(reqC)` for requests that are just ignored at the moment?

`chanDiscovery` requests are either ignored or received and never replied to (in `lib/reversetunnel/conn.go` and `lib/reversetunnel/agent.go`), `chanHeartbeat` is drained of requests but the requests are never replied to (in `lib/reversetunnel/agent.go` and in `lib/reversetunnel/localsite.go`/`lib/reversetunnel/remotesite.go`).

Reply:
I would prefer to address that in a separate PR.